LBM/Makefile

```diff
@@ -5,15 +5,16 @@
 CUDA_PATH ?= /usr/local/cuda-12.9
 
 ENZYME_PATH ?= $(HOME)/git/Reactant/enzyme/build/Enzyme/ClangEnzyme-22.so
-CLANG_PATH ?= $(HOME)/git/llvm-project/build/bin/clang++
-LIB_RAISE_PATH ?= $(HOME)/git/Enzyme-JAX/bazel-bin/libRaise.so
+CLANG_PATH ?= $(HOME)/git/Enzyme/llvm-project/buildRelease/bin/clang++
+LIB_RAISE_PATH ?= $(HOME)/git/Enzyme-JaX/bazel-bin/libRaise.so
 REACTANT_EXTRA_PATH ?= $(HOME)/git/Reactant.jl/deps/ReactantExtra/bazel-bin/
 
+export REACTANT_DEBUG=1
```

Comment:
Drop debug output when no longer needed? (For some reason cannot suggest changes through github)